i6_models/parts/frontend/protocol.py

```diff
@@ -0,0 +1,26 @@
+import torch
+import torch.nn as nn
+from abc import abstractmethod
+from typing import Protocol, Tuple
+
+
+class BaseFrontendInterface(Protocol):
```

Comment:
Should this also be versionized?
```suggestion
class BaseFrontendInterfaceV1(Protocol):
```
?